### ISSUE TITLE
Feature Suggestion: Team Management

### ISSUE THREAD

**Issue by secrethash** [bytefury/crater#29]
A Team Management setup for agencies would be a nice idea. One company; Many Team members; Different Roles.

**Comment by nfq** [comment]
Yeah, this was the first thing I looked at when testing the Demo this evening. I have a couple people on my team which would need to be able to login and create and edit Estimates, for instance.
